packages/lit-dev-content/samples/examples/motion-carousel/my-element.ts

```diff
@@ -0,0 +1,54 @@
+import {LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {animate} from '@lit-labs/motion';
+import {styleMap} from 'lit/directives/style-map.js';
+import {styles} from './styles.js';
+
+@customElement('my-element')
+export class MyElement extends LitElement {
+  static styles = styles;
+  @property({type: Number}) selected = 0;
+
+  data = Array(7).fill(null, 0);
+
+  render() {
+    return html`
+      <section class="container">
+        ${this.data.map((_v, i) => {
+          const l = this.data.length;
+          const c = Math.trunc(l / 2);
```

Comment:
Maybe some longer variable names here? Not too obvious what `l`, `c`, and `p` mean.

Reply:
Done.